### PR TITLE
Search page

### PR DESCRIPTION
**Pull request by liamjohnston**
Functionality already existed. I picked bits from the existing `cdhweb` directory and moved it into `templates/`. [As discussed](https://springload.slack.com/archives/C06JMUC4JBS/p1716254999506579).

Note, the radios don't do anything yet. Not sure if this requires some BED work... @haydngreatnews / @sarahframe? Noting that the existing site doesn't seem to have these filters on the search page.

New layout:
Screenshot 2024-05-21 at 3 29 17 PM

Each result has an icon:
Screenshot 2024-05-21 at 3 29 46 PM

There is some pagination:
Screenshot 2024-05-21 at 3 08 54 PM

### CODE REVIEW

File: cdhweb/static_src/global/components/footer.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .footer {
   --color-link-underline: var(--color-brand-110);
 
-  color: var(--color-primary-grey-110);
+  color: var(--color-grey-110);
```
Comment (liamjohnston) [inline cdhweb/static_src/global/components/footer.scss:4]: unrelated, fix for nonexistent colour name

File: cdhweb/static_src/styles.scss [hunk 2]
```diff
@@ -60,6 +61,7 @@
 @import './global/components/block-quote.scss';
 @import './global/components/download.scss';
 @import './global/components/search-form.scss';
+@import './global/components/pagination.scss';
```
Comment (liamjohnston) [inline cdhweb/static_src/styles.scss:64]: Kept pagination separate from the search page as it looks like other things will be paginated later on, e.g. news section

**Comment by sarahframe** [comment]
@liamjohnston yes bed work will be needed to make the filters work

File: templates/includes/pagination.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      class="pagination__item {% if number == page_obj.number %}pagination__item--current{% endif %}"
+      >
+      {{ number }}
+    </a>
+  {% endfor %}
+
+  {% if page_obj.has_next %}
+    <a rel="next" href="?{% url_replace 'page' page_obj.next_page_number %}" class="pagination__next-prev">
+      Next
+      {% include 'includes/svg.html' with sprite="two-tone" svg="chevron-right" classes="pagination__icon pagination__icon--prev" %}
```
Comment (phil-hodgson-1st) [inline templates/includes/pagination.html]: does this want to be `next-prev` to match the other class name or perhaps drop the prev and next and just have `icon`?

**Review by phil-hodgson-1st — APPROVED** [review]

**Comment by phil-hodgson-1st** [comment]
Awesome, looks good

File: templates/includes/pagination.html (hunk 3 above)
Comment (liamjohnston) [reply templates/includes/pagination.html]: It's the only icon, but sure, I can make that change

**Pull request merged**